cupy/core/elementwise.pxi

```diff
@@ -99,7 +99,10 @@ cdef dict _python_type_to_numpy_type = {
 
 
 cpdef _python_scalar_to_numpy_scalar(x):
-    if isinstance(x, six.integer_types):
+    # Note that isinstance(x, six_integer_types) matches with bool.
+    if isinstance(x, bool):
+        numpy_type = numpy.bool_
+    elif isinstance(x, six.integer_types):
```

Comment:
I think `type(x) in six.integer_types` is enough.

Reply:
Hmm, Is it worthwhile?
In my environment, both methods take almost exactly the same time (~93ns), and `isinstance` is more versatile and pythonic.